test_programs/plonky2_trace/3_two_files/expected_trace.json

```diff
@@ -0,0 +1,19 @@
+[
+  { "Path": "?" },
+  { "Function": { "path_id": 0, "line": -1, "name": "<toplevel>" } },
+  { "Call": { "function_id": 0, "args": [] } },
+  { "Type": { "kind": 30, "lang_type": "None", "specific_info": { "kind": "None" } } },
+  { "Path": "/home/stan/code/repos/noir/test_programs/plonky2_trace/3_two_files/src/main.nr" },
+  { "Step": { "path_id": 1, "line": 6 } },
+  { "Step": { "path_id": 1, "line": 7 } },
+  { "Path": "/home/stan/code/repos/noir/test_programs/plonky2_trace/3_two_files/src/foo.nr" },
+  { "Step": { "path_id": 2, "line": 3 } },
+  { "Step": { "path_id": 2, "line": 4 } },
+  { "Path": "/home/stan/code/repos/noir/test_programs/plonky2_trace/3_two_files/src/bar.nr" },
```

Comment:
me no likey

Reply:
Discussed offline and agreed on the current version with "abstract relative path".